Library/Homebrew/cask/installer.rb

```diff
@@ -136,10 +136,9 @@ def check_deprecate_disable
 
       case deprecate_disable_type
       when :deprecated
-        puts "::warning::#{message_full}" if ENV["GITHUB_ACTIONS"]
         opoo message_full
       when :disabled
-        puts "::error::#{message_full}" if ENV["GITHUB_ACTIONS"]
+        puts GitHub::Actions::Annotation.new(:error, message) if GitHub::Actions.env_set?
```

Comment:
Does it make sense for this function to do the `if`? It seems redundant to put it everywhere even though it would never work without GHA

Reply:
@SMillerDev Yeh, good point, refactoring into a helper method.